setup/files/supervisord.conf

```diff
@@ -5,13 +5,13 @@ pidfile=/opt/redash/supervisord/supervisord.pid
 directory=/opt/redash/current
 
 [inet_http_server]
-port = 127.0.0.1:9001
+port = 0.0.0.0:9001
```

Comment:
Well, I think it might be (generally) a better practice to bind to `0.0.0.0`. I mean, there is no reason not to do so :) Do you have further thinking on this?